internal/deviations/deviations.go

```diff
@@ -572,6 +572,12 @@ func QOSOctets(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosOctets()
 }
 
+// SetNativeUser creates a user and assigns role/rbac to that user via native model.
+func SetNativeUser(dut *ondatra.DUTDevice) bool {
+	logErrorIfFlagSet("deviation_set_native_user")
```

Comment:
Calling this function `logErrorIfFlagSet` is not required here since this is a new deviation and there is no flag called`deviation_set_native_user`.

This accesor can simply be:

```
func SetNativeUser(dut *ondatra.DUTDevice) bool {
	return lookupDUTDeviations(dut).GetSetNativeUser()
}
```

Reply:
Thanks @prinikasn , fixed in latest commit